core/src/main/scala/org/http4s/headers/Accept-Post.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2013 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s
+package headers
+
+import cats.data.NonEmptyList
+import cats.parse.Parser
+import org.http4s.internal.parsing.Rfc7230.headerRep1
+import org.typelevel.ci._
+//Accept-Post response header.
+//See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Post
+object `Accept-Post` {
+  def apply(head: MediaType, tail: MediaType*): `Accept-Post` = apply(
+    NonEmptyList(head, tail.toList))
+
+  def parse(s: String): ParseResult[`Accept-Post`] =
+    ParseResult.fromParser(parser, "Invalid Accept-Post header")(s)
+
+  private[http4s] val parser: Parser[`Accept-Post`] =
+    headerRep1(MediaType.parser).map(xs => `Accept-Post`(xs.head, xs.tail: _*))
+
+  implicit val headerInstance: Header[`Accept-Post`, Header.Recurring] =
+    Header.createRendered(
+      ci"Accept-Post",
+      _.values,
+      parse
+    )
+
+  implicit val headerSemigroupInstance: cats.Semigroup[`Accept-Post`] =
```

Comment:
Well, heck, you probably can even have a Monoid after the other comment!

Reply:
Can I assume there should be a comment about it maybe being able to just be a List inside?

I made it a Nel to match[ the Accept header](https://github.com/http4s/http4s/blob/series/0.22/core/src/main/scala/org/http4s/headers/Accept.scala#L81)

Reply:
The `Accept` header is wrong, too.  I think @bplommer fixed a couple of the CORS headers that made this mistake on 1.0, and you might find precedent there.

Reply:
@rossabaker I ported the changes made by @bplommer to the series/0.22. I tried to port it into the exact lines to make potential merges less painful. Let me know if there's any other changes you'd like to see.